src/NUnitEngine/nunit.engine.api/TestPackage.cs

```diff
@@ -49,7 +51,14 @@ public TestPackage(string filePath)
 
             if (filePath != null)
             {
+#if NETSTANDARD1_3
+                if (!Path.IsPathRooted(filePath))
```

Comment:
This is one major difference between the full engine and this version. The paths in .NET Standard vary dramatically depending on how you run the project. For example, it you use `dotnet run`, it compiles a project and runs it. The assembly is in a sub-directory, but the current directory is the location of the project file. If you use `dotnet execute` the path is that of the assembly. Because of this, our code that figures out the full path does not work in all cases, so I decided to only allow full paths and leave it up to the calling runner to resolve that path.